### PR TITLE
Focus section 6 on interaction between media transport protocols and …

### LINKED ISSUE #134: Comparison between UDP and TCP isn't quite right - should be between applications over UDP and over TCP
From @tfpauly

- As a bit of transport commentary, I don't find the setup of Section 6
compelling. While UDP is a transport in a technical sense, for the purpose of
media, it is almost always a layer upon which the protocol doing the congestion
control work runs. To that end, QUIC is just another more standardized case of
this, just like SCTP over UDP. Rather than talking about "UDP's behavior" vs
"TCP's behavior", I suggest talking about how applications over UDP for media
behave vs applications over TCP for media behave.

### PR DESCRIPTION
**Pull request by SpencerDawkins**
…transport protocols

Also tighten up description of the assumed protocol stack in (what is now) Section 6.1.

Note that the UDP and TCP sections have been flipped in order, to improve section flow and clarity.

closes #134

### CODE REVIEW

File: draft-ietf-mops-streaming-opcons.md [hunk 1]
```diff
@@ -716,45 +717,56 @@ To that effect, the Consumer Technology Association (CTA), who owns the Web Appl
 
 Many assume that the CDNs have a holistic view of the health and performance of the streaming clients. However, this is not the case. The CDNs produce millions of log lines per second across hundreds of thousands of clients and they have no concept of a "session" as a client would have, so CDNs are decoupled from the metrics the clients generate and report. A CDN cannot tell which request belongs to which playback session, the duration of any media object, the bitrate, or whether any of the clients have stalled and are rebuffering or are about to stall and will rebuffer. The consequence of this decoupling is that a CDN cannot prioritize delivery for when the client needs it most, prefetch content, or trigger alerts when the network itself may be underperforming. One approach to couple the CDN to the playback sessions is for the clients to communicate standardized media-relevant information to the CDNs while they are fetching data. {{CTA-5004}} was developed exactly for this purpose.
 
-# Evolution of Transport Protocols and Transport Protocol Behaviors {#sec-trans}
+# Iransport Protocol Behaviors and Their Effects on Media Senders and Receivers {#sec-trans}
```
Comment (GrumpyOldTroll) [inline draft-ietf-mops-streaming-opcons.md:720]: Iransport -> Transport

File: draft-ietf-mops-streaming-opcons.md [hunk 2]
```diff
@@ -716,45 +717,56 @@ To that effect, the Consumer Technology Association (CTA), who owns the Web Appl
...
 
-Because networking resources are shared between users, a good place to start our discussion is how contention between users, and mechanisms to resolve that contention in ways that are "fair" between users, impact streaming media users. These topics are closely tied to transport protocol behaviors.
+As described in the {{RFC8095}} Abstract, the IETF has standardized a number of protocols that provide transport services. Although these protocols, taken in total, provide a wide variety of transport services, this section will distinguish between
```
Comment (GrumpyOldTroll) [inline draft-ietf-mops-streaming-opcons.md:722]: Looks incomplete.  This ends with "distinguish between" ?  I think this is meant to finish with "Transport Protocols" and "Media Transport Protocols"?

**Review by GrumpyOldTroll — COMMENTED** [review]
Just a couple of nits here, but I think there might be more to say in another review

**Comment by SpencerDawkins** [comment]
@GrumpyOldTroll - 

> Just a couple of nits here, but I think there might be more to say in another review

If you have a moment before our call on Friday to take a look at this PR, I'd appreciate it. I think it's important to to get that right, in order to address both the INTDIR and TSVART reviews.

File: draft-ietf-mops-streaming-opcons.md [hunk 3]
```diff
@@ -716,45 +717,56 @@ To that effect, the Consumer Technology Association (CTA), who owns the Web Appl
...
+
+Modern TCP implementations ({{I-D.ietf-tcpm-rfc793bis}}) continue to probe for available bandwidth, and "back off" when a network path is saturated, but may also work to avoid growing queues along network paths, which prevent TCP senders from detecting quickly when a network path becoming saturated. Congestion control mechanisms such as COPA {{COPA18}} and BBR {{I-D.cardwell-iccrg-bbr-congestion-control}} make these decisions based on measured path delays, assuming that if the measured path delay is increasing, the sender is injecting packets onto the network path faster than the network can forward them (or the receiver can accept them) so the sender should adjust its sending rate accordingly.
```
Comment (GrumpyOldTroll) [inline draft-ietf-mops-streaming-opcons.md:763]: nit: "when a network path becoming saturated": missing an "is"?  Or should this be "has become"?

But also, I'm not sure it's right that avoiding growing queues prevents TCP senders from detecting this quickly.  (For most bottlenecks BBR will level off at saturation quicker than reno will stop growing cwnd past it, for instance.)

File: draft-ietf-mops-streaming-opcons.md (hunk 1 above)
Comment (SpencerDawkins) [reply draft-ietf-mops-streaming-opcons.md:720]: I've renamed this section a few times ...

File: draft-ietf-mops-streaming-opcons.md (hunk 2 above)
Comment (SpencerDawkins) [reply draft-ietf-mops-streaming-opcons.md:722]: I could reasonably have said "will distinguish between:" with a colon, but I went further, to be less confusing. This is now 

_As described in the {{RFC8095}} Abstract, the IETF has standardized a number of protocols that provide transport services. Although these protocols, taken in total, provide a wide variety of transport services, {{sec-trans}} will distinguish between two extremes:

- Transport protocols used to provide reliable, in-order media delivery to an endpoint, typically providing flow control and congestion control ({{reliable-behavior}}) and
- Transport protocols used to provide unreliable, unordered media delivery to an endpoint, without flow control or congestion control ({{unreliable-behavior}})._

File: draft-ietf-mops-streaming-opcons.md [hunk 4]
```diff
@@ -716,45 +717,56 @@ To that effect, the Consumer Technology Association (CTA), who owns the Web Appl
...
+- Transport protocols used to provide reliable, in-order delivery with flow control and congestion control, such as TCP ({{tcp-behavior}}) and
+- Transport protocols used to provide unreliable, unordered delivery without flow control or congestion control, such as UDP ({{udp-behavior}}).
 
-For most of the history of the Internet, these transport protocols, described in {{udp-behavior}} and {{tcp-behavior}}, have had relatively consistent behaviors that have changed slowly, if at all, over time. Newly standardized transport protocols like QUIC {{RFC9000}} can behave differently from existing transport protocols, and these behaviors may evolve over time more rapidly than currently-used transport protocols.
+We use "TCP" and "UDP" as shorthand for these classes of transport protocols because they have been so widely used on the Internet, but the reader should recognize that within {{sec-trans}}, they signify classes of transport protocols.
```
[the remark this answers is not in the document]
Comment (SpencerDawkins) [reply draft-ietf-mops-streaming-opcons.md:727]: @GrumpyOldTroll - I think you're half right, so I cut about half of the intro text. :grinning: I also combined this section and the following section.

File: draft-ietf-mops-streaming-opcons.md (hunk 3 above)
Comment (SpencerDawkins) [reply draft-ietf-mops-streaming-opcons.md:763]: Thanks for asking this question - I meant **OLDER** TCP implementations, but didn't say that.

File: draft-ietf-mops-streaming-opcons.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (GrumpyOldTroll) [reply draft-ietf-mops-streaming-opcons.md:727]: works for me :)

**Pull request merged**